test/testcafe/spec/EnrollAuthenticatorDataPhoneView_spec.js

```diff
@@ -44,7 +44,7 @@ test.requestHooks(mock)('default sms mode', async t => {
   // Extension field is hidden
   await t.expect(enrollPhonePage.extensionIsHidden()).eql(true);
 
-  await t.expect(await enrollPhonePage.signoutLinkExists()).notOk();
+  await t.expect(await enrollPhonePage.signoutLinkExists()).ok();
```

Comment:
How/why did this change? I thought we are only changing the text...

@vincentvoong-okta - So this is the screen in Enroll Phone flow where we have the radio options for users to select between voice and sms and enter their phone number (with extension for voice). With this change, we will have a `Back to sign in` link in the footer, is this ok?

Reply:
I think the JIRA lists the places where we want the `back to sign in` link https://oktainc.atlassian.net/browse/OKTA-381328?focusedCommentId=1873997